public/styles/components/_hover-expander.scss

```diff
@@ -0,0 +1,26 @@
+.hover-expander {
+  position: relative;
```

Comment:
Let's add
```suggestion
  position: relative;
  contain: size style layout;
```

Reply:
neat, yeah

Reply:
hey, I had to take out `size`. This is because the element doesn't have an intrinsic size (it depends on the children) so it was rendering at `0x0px`. This sounds like expected behaviour to me so I just removed it